README.md

```diff
@@ -36,4 +38,10 @@ https://speakerdeck.com/mbazaliy/jailbreaking-apple-watch-1
 
 https://speakerdeck.com/mbazaliy/jailbreaking-apple-watch
 
+### Compatibility
+Because of the way that I (fr3x31dev) implemented the FS patch, it rules out any support on watchOS 4.x. This can be improved through some other function (can't remember it off the top of my head, I focus on pre-iOS 11 development) that Apple **DIDN'T** patch, making this able to work until watchOS 4.1 (provided exploits, and other patches) I don't know how feasiable this is, if anyone actually uses it.
```

Comment:
- I highly doubt apple did patch out fstab on iOS and didn't on watchOS
- Even if you remount root, it won't be writeable unless it's apfs or you patch lwvm, see [this](https://stek29.rocks/2018/01/22/lwvm-mapforio.html)